test/stability/testCksum/testCksum.go

```diff
@@ -307,7 +307,7 @@ func composeStatistics() error {
 	println("Sent", sent, "packets")
 	println("Received", received, "packets")
 	println("Ratio = ", ratio, "%")
-	if atomic.LoadInt32(&passed) != 0 && ratio >= 90 {
+	if atomic.LoadInt32(&passed) != 0 && ratio >= 98 {
```

Comment:
Can you use a named constant defined at the beginning of the file instead of magic number 98?